Academy2/Academy2/Car.cs

```diff
@@ -1,10 +1,12 @@
 ﻿using System;
+using Academy2.Enums;
 
 namespace Academy2
 {
     public class Car
     {
         public Guid Id { get; set; }
+        public Brand Brand { get; set; }
         public string Model { get; set; }
         public int Doors { get; set; }
```

Comment:
```suggestion
        public byte Doors { get; set; }
```
 se podria cambiar  int por byte